### PR TITLE
Adds hierophant antenna traitor item

### PR DESCRIPTION
**Pull request by Takahiru**
Adds a 14 TC hierophant antenna traitor item, allows traitor shaft miners to use the hierophant club on the station.

### CODE REVIEW

**Comment by Takahiru** [comment]
oh no what is merge conflict

**Comment by Alexandria404** [comment]
very nice

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -1372,8 +1372,25 @@
...
+
+/obj/item/hierophant_club/attackby(obj/item/I, mob/user)
+	if(istype(I, /obj/item/hierophant_antenna) && z_level_check == TRUE)
+		z_level_check = FALSE
+		desc += " It has an ominous antenna attached."
+		qdel(I)
+	else
+		if(istype(I, /obj/item/hierophant_antenna) && z_level_check == FALSE)
+			to_chat(user, "<span class='warning'>The herald's power already reaches this club!</span>")
+
```
Comment (alexkar598) [inline code/modules/mining/lavaland/necropolis_chests.dm]:
```suggestion
/obj/item/hierophant_club/attackby(obj/item/I, mob/user)
	if(istype(I, /obj/item/hierophant_antenna))
		if(z_level_check)
			z_level_check = FALSE
			desc += " It has an ominous antenna attached."
			qdel(I)
		else
			to_chat(user, "<span class='warning'>The herald's power already reaches this club!</span>")
		return TRUE
	else
		return ..()

```

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: code/modules/mining/lavaland/necropolis_chests.dm (hunk 1 above)
Comment (alexkar598) [reply code/modules/mining/lavaland/necropolis_chests.dm]: That brings slightly cleaner code and proper parent calling

**Pull request merged**